conda.recipe/bld.bat

```diff
@@ -9,3 +9,9 @@ if errorlevel 1 exit 1
 
 copy %SRC_DIR%\shell\conda.fish %PREFIX%\etc\fish\conf.d
 if errorlevel 1 exit 1
+
+mkdir %SP_DIR%\conda\resources
+if errorlevel 1 exit 1
+
+copy %SRC_DIR%\conda\resources\* %SP_DIR%\conda\resources
```

Comment:
@mingwandroid Can you confirm this `copy` command is going to work as I expect it to? That is, copy all contents of that recipe resources directory into the new resources directory...